pages/pipelines/artifacts.md.erb

```diff
@@ -153,3 +153,26 @@ Alternatively, you can use a self-managed storage provider. Read these guides fo
 If you manage your own artifact storage, then you are responsible for encryption and retention planning.
 
 To track the actions of users with access to your artifacts, use the [API Access Audit](https://buildkite.com/organizations/~/api-access-audit).
+
+## Troubleshooting: `Multiple artifacts were found for query` error
+
+### Message
```

Comment:
```suggestion
## Troubleshooting for `Multiple artifacts were found for query` error

If you encounter the following error:
```

Reply:
I have 2 issues with the initial structure:
- Everywhere in the docs there is usually at least some lead-in text between a H2 and H3
- Let's make the whole entry a bit more human-friendly.

Reply:
OK, let me try this again without the headings. I optimized for skimming here, but if we're going to go with introductory text like this, then I think it'll work better without any of the headings. Expect a bit of a do-over here soon.